workers/unity/Packages/io.improbable.gdk.gameobjectcreation/Tests/Editmode/GameObjectCreationTests.cs

```diff
@@ -38,15 +38,11 @@ public new void Setup()
             linker = goInitSystem.Linker;
         }
 
-        [TearDown]
-        public new void TearDown()
-        {
-            World.Dispose();
```

Comment:
do we not need to dispose of the world anymore? ah, is this handled by the refactor jamie did a while ago?

Reply:
We do, but it's already being done in the base class